python/paddle/fluid/layers/detection.py

```diff
@@ -909,3 +910,95 @@ def _is_list_or_tuple_and_equal(data, length, err_info):
     box.stop_gradient = True
     var.stop_gradient = True
     return mbox_locs_concat, mbox_confs_concat, box, var
+
+
+def anchor_generator(input,
```

Comment:
Need unit testing for the interface in https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/test_detection.py

Reply:
revised